<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1362</width>
    <height>782</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Centro</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Norte</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sul</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tupy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vila Nova</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Iririu</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Itaum</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nova Brasilia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Guanabara</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pirabeiraba</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Campus</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Inicio</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Destino</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular Rota</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>420</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Centro</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Norte</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sul</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Tupy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vila Nova</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Iririu</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Itaum</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nova Brasilia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Guanabara</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pirabeiraba</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Campus</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>411</width>
      <height>221</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Zarco.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>791</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Mapa Centro.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_centro">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>791</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Centro/CENTRO.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_pirabeiraba">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>801</width>
      <height>661</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Pirabeiraba/PIRABEIRABA.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_sul">
    <property name="geometry">
     <rect>
      <x>538</x>
      <y>13</y>
      <width>801</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Sul/SUL.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_norte">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>791</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Norte/NORTE.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_nortesul">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>791</width>
      <height>701</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Norte/NORTE - SUL.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_nortepirabeiraba">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>791</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Norte/NORTE-PIRABEIRABA.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_vilanova">
    <property name="geometry">
     <rect>
      <x>548</x>
      <y>13</y>
      <width>791</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Vila Nova/Vila Nova.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_itaum">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>791</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Itaum/Itaum.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_novabrasilia">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>791</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Nova Brasilia/Nova Brasilia.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_guanabara">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>791</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Guanabara/Guanabara.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_campus">
    <property name="geometry">
     <rect>
      <x>548</x>
      <y>13</y>
      <width>791</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Campus/Campus.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_iririu">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>791</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Iririu/Iririu.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_tupy">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>781</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Tupy/Tupy.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_centrosul">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>-10</y>
      <width>801</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Centro/centro-sul.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_centronorte">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>-10</y>
      <width>811</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Centro/Centro-norte.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_centroitaum">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>801</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Centro/centro-itaum.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_centrotupy">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>801</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Centro/centro-tupy.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_centrocampus">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>801</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Centro/centro-campus.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_centrovilanova">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>791</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Centro/centro-vilanova.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_centronovabrasilia">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>791</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Centro/centro-novabrasilia.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_centroiririu">
    <property name="geometry">
     <rect>
      <x>538</x>
      <y>3</y>
      <width>801</width>
      <height>691</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Centro/centro-iriru.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_centroguanabara">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>801</width>
      <height>671</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Centro/centro-guanabara.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_distancia">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>460</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_distancianum">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>460</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_nortevilanova">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>801</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Norte/VilaNova-Norte.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_norteiririu">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>791</width>
      <height>661</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Norte/Iririu-Norte.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_tupyiririu">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>801</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Tupy/Iririu-Tupy.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_tupyguanabara">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>791</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Tupy/Guanabara-Tupy.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_sulitaum">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>791</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Sul/Itaum-Sul.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_campussul">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>791</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Campus/Sul-Campus.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_campusnorte">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>801</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Campus/Norte-Campus.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_campuscentro">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>801</width>
      <height>671</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Campus/Centro-Campus.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_campusitaum">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>791</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Campus/Itaum-Campus.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_campusiririu">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>791</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Campus/Iririu-Campus.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_pirabeirabavilanova">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>801</width>
      <height>691</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/new/prefix1/Mapa/Pirabeiraba/Pirabeiraba-Vilanova.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>460</y>
      <width>59</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>metros</string>
    </property>
   </widget>
   <zorder>comboBox</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_4</zorder>
   <zorder>label</zorder>
   <zorder>comboBox_2</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_pirabeiraba</zorder>
   <zorder>label_nortesul</zorder>
   <zorder>label_sul</zorder>
   <zorder>label_nortepirabeiraba</zorder>
   <zorder>label_novabrasilia</zorder>
   <zorder>label_tupy</zorder>
   <zorder>label_centrosul</zorder>
   <zorder>label_centroitaum</zorder>
   <zorder>label_centrotupy</zorder>
   <zorder>label_centrocampus</zorder>
   <zorder>label_centrovilanova</zorder>
   <zorder>label_centronovabrasilia</zorder>
   <zorder>label_centroiririu</zorder>
   <zorder>label_centronorte</zorder>
   <zorder>label_centroguanabara</zorder>
   <zorder>label_distancia</zorder>
   <zorder>label_distancianum</zorder>
   <zorder>label_nortevilanova</zorder>
   <zorder>label_norteiririu</zorder>
   <zorder>label_tupyiririu</zorder>
   <zorder>label_tupyguanabara</zorder>
   <zorder>label_iririu</zorder>
   <zorder>label_guanabara</zorder>
   <zorder>label_sulitaum</zorder>
   <zorder>label_campussul</zorder>
   <zorder>label_campus</zorder>
   <zorder>label_campusnorte</zorder>
   <zorder>label_centro</zorder>
   <zorder>label_campuscentro</zorder>
   <zorder>label_campusitaum</zorder>
   <zorder>label_itaum</zorder>
   <zorder>label_campusiririu</zorder>
   <zorder>label_pirabeirabavilanova</zorder>
   <zorder>label_vilanova</zorder>
   <zorder>label_5</zorder>
   <zorder>label_norte</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
